java/memory/src/main/java/org/apache/arrow/memory/AllocationManager.java

```diff
@@ -53,7 +53,7 @@
   private static final PooledByteBufAllocatorL INNER_ALLOCATOR = new PooledByteBufAllocatorL();
 
   static final UnsafeDirectLittleEndian EMPTY = INNER_ALLOCATOR.empty;
-  static final long CHUNK_SIZE = INNER_ALLOCATOR.getChunkSize();
+  public static final long CHUNK_SIZE = INNER_ALLOCATOR.getChunkSize();
```

Comment:
maybe pass this in as a parameter to the policy, instead of making it public?

Reply:
Since the default rounding policy is frequently used, for performance considerations, I make it a singleton which is shared by all allocators. 
If we make the chunk size as the parameter, maybe we will need to create a default policy for each allocator.

In the new commit, I get the chunk size through reflection, and restore the change here.